src/icalendar/cal/component.py

```diff
@@ -20,6 +20,35 @@
 _marker = []
 
 
+def _infer_value_type(value):
```

Comment:
Please add a type hint here.
```suggestion
def _infer_value_type(value:date|datetime|timedelta|time|tuple|list) -> str|None:
```